packages/client/integration/ProtectionRecovery.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { buildApi } from '@logion/node-api';
+import { Keyring } from '@polkadot/api';
+import { UUID } from '@logion/node-api/dist/UUID';
+
+import { LogionClient } from '../src/';
+import { AxiosFactory } from '../src/AxiosFactory';
+import { AcceptedProtection, ActiveProtection, NoProtection, PendingProtection, PendingRecovery } from '../src/Recovery';
+import { LogionClientConfig } from '../src/SharedClient';
+import { KeyringSigner, FullSigner, Signer } from '../src/Signer';
+import { LegalOfficer } from '../src/Types';
+
+import { ALICE, BOB } from '../test/Utils';
+import { ProtectionRequest } from '../src/RecoveryClient';
+
+describe("Recovery SDK", () => {
+
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+
+    it("enables protection", async () => {
+        const config = {
+            directoryEndpoint: "http://localhost:8090",
+            rpcEndpoints: [ 'ws://localhost:9944', 'ws://localhost:9945' ]
+        };
+        const client = await LogionClient.create(config);
+        const signer = buildSigner();
+        const authenticatedClient = await client.authenticate([ REQUESTER_ADDRESS ], signer);
+
+        const legalOfficers = await client.getLegalOfficers();
+        const alice = legalOfficers.find(legalOfficer => legalOfficer.address === ALICE.address)!;
+        const bob = legalOfficers.find(legalOfficer => legalOfficer.address === BOB.address)!;
+
+        const noProtection = await authenticatedClient.protectionState() as NoProtection;
+        const pending = await noProtection.requestProtection({
+            legalOfficer1: alice,
+            legalOfficer2: bob,
+            userIdentity: {
+                email: "john.doe@invalid.domain",
+                firstName: "John",
+                lastName: "Doe",
+                phoneNumber: "+1234",
+            },
+            postalAddress: {
+                city: "",
+                country: "",
+                line1: "",
+                line2: "",
+                postalCode: "",
+            }
+        });
+
+        await acceptRequest(config, client, signer, alice);
+        await transferTokens(config, signer, alice.address, REQUESTER_ADDRESS, 1000000000);
```

Comment:
This transfer is done to ensure that requester can pay the fees of activation, right ?
When reading this sequence one could think it's part of the flow of protection.
In this case I would move this call to the beginning of the test, and also name the method `initBalance()` or `ensureMinimalBalance()` or something like this.

Reply:
Good point.